### PR TITLE
Add responsiveResize to charts

### PR DESCRIPTION
**Pull request by steven-ferguson**
Adds responsive resize discussed in #1

### CODE REVIEW

File: config/environment.js [hunk 1]
```diff
@@ -1,5 +1,15 @@
...
-  return { };
+  /*
+    Have to provide ember-resize defaults due to https://github.com/mike-north/ember-resize/issues/23
+  */
+  return {
+    resizeServiceDefaults: {
+      debounceTimeout    : 200,
+      heightSensitive    : true,
+      widthSensitive     : true,
+      injectionFactories : [ 'view', 'component'],
```
Comment (sir-dunxalot) [inline config/environment.js]: Will including `view` here thrown a warning for newer Ember builds?

Comment (steven-ferguson) [inline config/environment.js]: I've been running this on Ember 2.5.x and have not seen any warnings. I also don't see any [ember-resize issues](https://github.com/mike-north/ember-resize/issues?q=is%3Aopen+is%3Aissue) about it.

Comment (sir-dunxalot) [inline config/environment.js]: Ok, sounds good!

**Comment by sir-dunxalot** [comment]
This looks good! Is there a relatively easy way to include a test for the resize feature?

**Comment by steven-ferguson** [comment]
We could trigger a resize event ([example](https://github.com/mike-north/ember-resize/blob/v0.0.15/tests/unit/services/resize-test.js#L20-L22)) and then assert that `chartDidRender` was fired.

**Comment by sir-dunxalot** [comment]
Sounds good! Do you mind adding a test that will fail without this feature? Once that's up I'm more than happy to merge and release!

**Comment by steven-ferguson** [comment]
For this to work correctly on resizing to a smaller window size, the chart needs either a max-width or width set otherwise the auto sizing will rerender the chart with the same width it had on the first render. I think there are two ways we could solve this:

1) Add a default max-width to `.google-chart` based on the viewport width:
```css
.google-chart {
  max-width: 95vw;
}
```

2) In the resize handler on the `google-chart` component, get the current width and add it to the default options:
```javascript
resizeChart() {
  const currentWidth = window.innerWidth;
  this.set('defaultOptions.chartArea.width', currentWidth);
  this._rerenderChart();
}
```

In my project,  I am using the max-width approach which is working well, but the downside is that on mobile the charts will shrink down to fit the screen size.

**Comment by sir-dunxalot** [comment]
Hey Steven, were you planning on making any changes after your last comment or are we all good here? I can get this merged and released soon.

**Comment by steven-ferguson** [comment]
I still need to make the change that gives the chart a width, so that it will trigger a resize when the window size is reduced to below the size of the original container.

**Comment by sir-dunxalot** [comment]
Ok, no rush! Just checking.

**Comment by steven-ferguson** [comment]
Closing in favor of #28

**Pull request closed without merging**